src/main/java/com/hw/bookstore/dto/BookRequestDto.java

```diff
@@ -0,0 +1,32 @@
+package com.hw.bookstore.dto;
+
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import java.math.BigDecimal;
+
+public record BookRequestDto(
+
+        @NotNull(message = "Title can not be null")
+        @Size(min = 1, max = 255, message = "Title size must be between 1 and 255 symbols")
+        String title,
+
+        @NotNull(message = "Author can not be null")
+        @Size(min = 1, max = 255, message = "Author size must be between 1 and 255 symbols")
+        String author,
+
+        @NotNull(message = "Isbn can not be null")
+        @Size(min = 1, max = 255, message = "Isbn size must be between 1 and 255 symbols")
+        String isbn,

```

Comment:
You can add specific Isbn validation (use pattern or create your own annotation)

Reply:
That's great advice, I'll think about it.